Modelica/Mechanics/Rotational/Components/Clutch2.mo

```diff
@@ -7,11 +7,11 @@ model Clutch2 "Clutch based on Coulomb friction (with interpolation by ExternalC
   import TabInternal = Modelica.Blocks.Tables.Internal;
 
   parameter Real mu_pos[:, 2] = [0, 0.5]
-    "Positive sliding friction coefficient [-] as function of w_rel [rad/s] (w_rel>=0)";
+    "Positive sliding friction coefficient [-] as function of w_rel [rad/s] (w_rel >= 0)";
```

Comment:
I'd actually avoid to introduce cosmetical changes now. This should go to a separate issue addressing all component models.

Reply:
Ok, how to revert this?

Reply:
> 
> 
> Ok, how to revert this?

```
git reset ca8078fbd49f1f9ab892818bbcd93a3d60ac12f1
git push --force
```

Reply:
Done. The first time I've ever did it!

Reply:
You still have the changes locally. So be careful when pushing again.

Reply:
I use TortoiseSVN which enables three options how to reset a commit. I selected the one where also local changes are reset.
But anyway, thanks for the hint because I've never done it so far.

Reply:
I truly hope you are **not** using Tprotois**SVN** in order to work on the git repo ;-)